http/1.6/modules/ROOT/pages/http-authentication.adoc

```diff
@@ -1023,9 +978,11 @@ When using multiple RO with a single Token Manager, if you want to only clear th
 </flow>
 ----
 
-=== Token Manager - Customize the Token Manager Object Store
+=== Configure Token Manager Object Store
+
+By default, the token manager uses an in-memory object store to store the credentials. You can configure the token manager object store by using the *Object store* field.
 
-By default, the token manager uses an in-memory object store to store the credentials. You can customize the token manager object store by using the `objectStore` attribute. See also xref:object-store-connector::object-store-to-define-a-new-os.adoc[Configure a custom object store].
+For further information about object stores, refer to the xref:object-store-connector::object-store-to-define-a-new-os.adoc[Configure a custom object store] documentation.
```

Comment:
For more info about object stores, see the information about configuring a custom object store? The reference is to a very specific thing about object stores, rather than just "more information" in general.

Reply:
Sentence lettercasing for the xref breaks up the sentence the xref is in, so I think you want...:
```suggestion
For further information about object stores, refer to the documentation about xref:object-store-connector::object-store-to-define-a-new-os.adoc[configuring a custom object store].
```